.travis.yml

```
@@ -27,6 +27,7 @@ before_script:
       sudo apt-get -qq update;
       sudo apt-get install -y shellcheck;
     fi
+  - test "$SPRING_PROFILES_ACTIVE" = "travis" && pip install --user ansible-lint || true
```

What's the point to put it separately given that we already has `if [ "$SPRING_PROFILES_ACTIVE" = 'travis' ]; then`? I'd rather put all installation commands together. Especially because order of `pip` commands matter as it may lead to different versions of the dependencies will be installed.

Also, we shouldn't hide/ignore error with `|| true`. Let it fail so it won't go unnoticed.

---

Fixed